var/spack/repos/builtin/packages/py-qtpy/package.py

```diff
@@ -0,0 +1,36 @@
+##############################################################################
+# Copyright (c) 2013-2016, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/llnl/spack
+# Please also see the LICENSE file for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class PyQtpy(PythonPackage):
+    """QtPy: Abtraction layer for PyQt5/PyQt4/PySide"""
+
+    homepage = "https://github.com/spyder-ide/qtpy"
+    url      = "https://files.pythonhosted.org/packages/0a/a8/31632a1c7cab273ac801da8b610d8659195853c099e1a26bfc08e78ddbe6/QtPy-1.2.1.tar.gz"
+
+    version('1.2.1', 'e2f783fb7f8e502815237bd8d30c6d11')
+
+    depends_on('py-setuptools', type='build')
```

Comment:
Looks like this is missing some dependencies. From the [homepage](https://github.com/spyder-ide/qtpy):
```
Requirements

You need PyQt5, PyQt4 or PySide installed in your system to make use of QtPy. If several of these packages are found, PyQt5 is used by default unless you set the QT_API environment variable.

QT_API can take the following values:

    pyqt5 (to use PyQt5).
    pyqt or pyqt4 (to use PyQt4).
    pyside (to use PySide).
```